profiles/notifications.py

```diff
@@ -0,0 +1,7 @@
+from django.utils.translation import ugettext_lazy as _
+from django_ilmoitin.registry import notifications
+
+notifications.register(
+    "relationship_confirmation_needed", _("Legal relationship confirmation needed")
```

Comment:
can we store the code strings as enum values? will be easier to change the Enum class and not change every place where the code string is used.